README.md

```diff
@@ -12,31 +12,46 @@
 
 ## Supported Languages
 
-'Dependency Analytics' extension supports projects using Maven, projects build on npm (Node ecosystem).
-Extending support for Python and Go languages is currently under progress.
+'Dependency Analytics' extension supports projects using Maven, projects build on npm (Node ecosystem) and projects using Python.
+Extending support for Golang and other languages is currently under progress.
```

Comment:
Do we need to say `other languages`? isn't it very generic statement for extension READMEs